### DISCUSSION TITLE
limit_execution error but seems to work

### DISCUSSION THREAD

**Discussion by vibarco** [jertel/elastalert2#687] [Q&A] [answered]
Hello,

when i try this parameter i recieved this error but the rule seems to execute correctly only between that range:

`limit_execution: "* 22-6 * * *"`

`croniter.croniter.CroniterBadCronError: [* 22-6 * * *] is not acceptable`

Best

**Comment by nsano-rururu** [comment]
limit_execution uses a python library called croniter. It seems that some error occurred in the specified value in croniter.
https://github.com/kiorky/croniter

**Comment by jertel** [comment, marked as answer]
You cannot use decreasing range values. Instead use multiple ranges: `* 22-23,0-6 * * *`

Use this web tool to validate your cron config: https://crontab.guru/#*_22-23,0-6_*_*_*
